Source/WebCore/page/Navigation.cpp

```diff
@@ -912,7 +912,7 @@ Navigation::DispatchResult Navigation::innerDispatchNavigateEvent(NavigationNavi
         { false, canBeCanceled, false },
         navigationType,
         destination.ptr(),
-        abortController->protectedSignal(),
+        Ref { abortController->signal() },
```

Comment:
Do we really need the `Ref { }` here?

Reply:
Yes, for this one and the two you flagged below I could not get it to compile without the explicit wrapper unfortunately. That's why I kept around the protected method initially.